symfony.lock

```diff
@@ -86,6 +86,9 @@
     "kevinrob/guzzle-cache-middleware": {
         "version": "v3.3.1"
     },
+    "league/csv": {
+        "version": "9.8.0"
```

Comment:
Do you need to pin to this exact version?

Reply:
I guess that's fine, but I was wondering why you would not use `^` here. Dane or Matt can chime in and advise, I guess 😅

Reply:
Ahhh I see - I'm happy to put in a `^` there, whatever would work best!

Reply:
Yes. Use `^`.